src/core/network/getRequestRetryDelay.js

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { RETRY_AFTER } from "../../constants/httpHeaderNames";
+import { isIntegerString } from "../../utils";
+
+// The retry gets incrementally (but not exponentially) longer for each retry.
+const FIRST_DELAY_MILLIS = 1000;
+const INCREMENTAL_DELAY_MILLIS = 1000;
+
+// When the target delay is randomized, make it within the range of this percentage above or below the target delay.
+const MAX_RANDOM_VARIANCE_PERCENTAGE = 0.3;
```

Comment:
@raresvla, you might want to provide input on these three constants.

Reply:
👍 The values look good to me as they are set right now.